doc/man/pgagroal_databases.conf.5.rst

```diff
@@ -31,16 +32,40 @@ INITIAL_SIZE
 MIN_SIZE
   Specifies the minimum pool size for the entry. Default is 0. Requires a pgagroal_users.conf configuration
 
+DATABASE ALIASES
+================
+
+Database aliases allow clients to connect using alternative names for a configured database. This is useful for application migrations, multi-tenancy, or providing user-friendly names.
+
+Aliases are specified in the DATABASE field using the format: database_name=alias1,alias2,alias3
+
+ALIAS RULES:
+  - Maximum 8 aliases per database
+  - Aliases must be globally unique across all database entries
+  - Alias names cannot conflict with any real database name
+  - Database names cannot conflict with any alias name
+  - The special database name "all" cannot have aliases
+  - Empty alias names are not allowed
+
+When a client connects using an alias, pgagroal transparently maps it to the real database name for all backend operations.
+
 EXAMPLE
 =======
 
 ::
    
   #
-  # DATABASE USER  MAX_SIZE INITIAL_SIZE MIN_SIZE
+  # DATABASE                    USER    MAX_SIZE INITIAL_SIZE MIN_SIZE
```

Comment:
can we change the header into
`# DATABASE [=alias, alias, ...]      USER MAX_SIZE INITIAL_SIZE MIN_SIZE`